### ISSUE TITLE
Create Studio

### ISSUE THREAD

**Issue by SpyC0der77** [TimMcCool/scratchattach#277]
Is there any way to create a studio using the python? That might have to be a part of scratchlenium or whatever it was called

**Comment by FAReTek1** [comment]
i'll take a look

**Comment by FAReTek1** [comment]
yeah, the url is [https://scratch.mit.edu/studios/create/](). I'll make a function for it then make a pr

**Comment by SpyC0der77** [comment]
network tab coming in clutch :)

**Comment by FAReTek1** [comment]
hmm. chrome isnt telling me about any payload. also, whenever i try to make a request, it spits back a non-200 status code.

**Comment by FAReTek1** [comment]
oops, looks like the account i was using to try and make a studio, causing issues. Should make an error for that

**Comment by SpyC0der77** [comment]
i don't think there should be a payload. just cookies and headers. because it is just supposed to create one with no name or contributers

**Comment by FAReTek1** [comment]
lmao, yeah it works now, i was using the wrong account lol

**Comment by FAReTek1** [comment]
> i don't think there should be a payload. just cookies and headers. because it is just supposed to create one with no name or contributers

yeah that's what i thought, but when i was getting errors i didnt know what else could possibly cause issues

**Comment by FAReTek1** [comment]
ill make options to set title etc so you only need to make 1 function call. I'll also add a check for new scratchers

**Comment by FAReTek1** [comment]
yep, done! I'll test it, then make a pr

**Comment by FAReTek1** [comment]
there should be a rate limit on it like session.create project

**Comment by SpyC0der77** [comment]
yeah

**Comment by SpyC0der77** [comment]
yay

**Comment by FAReTek1** [comment]
I will make the pr i guess now. also found a typo in cloud_requests so i fixed it

**Comment by FAReTek1** [comment]
[pull req that adds this feature](https://github.com/TimMcCool/scratchattach/pull/279)

**Comment by FAReTek1** [comment]
unfortunately it still hasnt been merged into the main scratchattach module xD

**Comment by SpyC0der77** [comment]
LOL
